src/js/helpers/reviewHelper.js

```diff
@@ -172,7 +172,7 @@ const getFileStates = (status) => {
 		else {
 			let count = 0;
 			item.error_data.forEach((error) => {
-				count += parseInt(error.occurrences);
+				count += parseInt(error.occurrences, 10);
```

Comment:
The `10` here is to ensure the result is in base 10. But that should be the default (provided that the number in question doesn't start with a 0), no? 
Is it possible for `error.occurrences` to start with a 0?